source/chapters/appendix/mixxx_controls.rst

```diff
@@ -752,6 +752,58 @@ Any control listed above for :mixxx:cogroupref:`[ChannelN]` will work for a samp
    .. versionadded:: 2.0.0
 
 
+.. mixxx:control:: [ChannelN],shift_cues_earlier
+                   [PreviewDeckN],shift_cues_earlier
+                   [SamplerN],shift_cues_earlier
+
+   :range: binary
+   :feedback: The :term:`hotcues <hotcue>` move left by 10ms.
```

Comment:
Note that this moves **all** cues. IIRC the intention is to compensate for decoder offsets.
https://github.com/mixxxdj/mixxx/blob/9b31059fbacdd2d66bcf397b2354bb32c7ad1872/src/track/track.cpp#L978